CodeStarterCamp_Week1/Step3.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  second.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Jeong Min Lee on 2/13/24.
+//
+
+import Foundation
+
+func drawPepero(bodyWidth width: Int,pockyBody body: String ,peperoTopping topping: String, barLength length : Int) {
+    print ("""
+            <정보>
+            길이: \(width)
+            몸통: \(body)
+            토핑: \(topping)
+            막대길이: \(length)
+            """)
+    drawPeperoBody(width: width, body: body, topping: topping)
+    drawPeperoBar(length: length)
+}
+
+func drawPeperoBody(width: Int, body: String, topping: String) {
+    for _ in 1...width {
+        for _ in 1...1 {
+            print("",topping, terminator: "")
+        }
+        print(body, topping, separator: "")
+    }
+    
+}
+
+func drawPeperoBar(length: Int) {
+    for _ in 1...length {
+        for _ in 1...2 {
+            print(" ", terminator: "")
+        }
+        print("| |")
```

Comment:
```suggestion
        print(" | |")
```

이 부분도 무조건 2 칸을 띄우기 위해 사용된 코드인 것 같습니다. 토핑의 길이 등을 입력받는다면 경우가 다를 수도 있지만 이 상황에서 꼭 필요한 코드로는 보이지 않는데, 혹시 어떤 이유에서 작성한 코드인지 알 수 있을까요?

Reply:
2칸을 띄운다는 생각 때문에 의식하지않고 작성했던 것 같습니다.